cmd/http-tracer.go

```diff
@@ -227,7 +149,7 @@ func Trace(f http.HandlerFunc, logBody bool, w http.ResponseWriter, r *http.Requ
 	t.CallStats = trace.CallStats{
 		Latency:     rs.Time.Sub(rq.Time),
```

Comment:
this can also be calculated using rs.Time.Sub(rw.startTime) which may be a bit more accurate...